deploy/scripts/src/authentication/test.ts

```diff
@@ -386,11 +366,75 @@ export function signIn(): void {
       isStatusCode302
     })
     // 03_AuthCall
-    res = timeGroup(groups[3].split('::')[1], () => http.get(res.headers.Location), {
+    return (res = timeGroup(
```

Comment:
```suggestion
    return timeGroup(
```
Return this directly, the assignment will not do anything